app/controllers/add_task_controller.rb

```diff
@@ -1,6 +1,6 @@
 class AddTaskController < ApplicationController
```

Comment:
Per rails conventions, it would make more sense to use one single controller called "TasksController" which would take care of all of the add, display and update operations.